lips/lip-6.mdx

```diff
@@ -10,27 +10,26 @@ created: 07/30/2020
-* *Reconfiguration* defines a new epoch by triggering a new epoch event (`LibraConfig::NewEpochEvent`), it gets emitted if any part of the global on-chain configuration, such as the Validator Set, changes. If a transaction causes an epoch change, that transaction is the last transaction committed in the current epoch — any subsequent transactions in that block or future blocks from that epoch will be ignored. Once the transaction has been committed, the new configuration starts being valid immediately. E.g. the new Validator Set will start running the consensus protocol in the new epoch.
-* *LibraRoot* is an account that may add/remove members to/from the ValidatorSet, it may also send an AdminScript.
+* *Validator Operator* is an owner of an on-chain Libra account, it may operate none, one or several Validator Nodes: a Validator Owner may designate a Validator Operator to run a Validator Node on Validator Owner's behalf. A Validator Operator may run multiple Validator Nodes. The Validator Operator for a Validator Owner has permission to modify the Validator Config stored in the Validator Owner account and update its copy in the Validator Set.
+* *Validator Set* is an on-chain Move resource represented by a vector containing the active set of Validator Nodes. There exists a one-to-one relationship between a Validator Owner in the Validator Set and a Validator Node. The entries within the Validator Set consist of the Validator Owner's address as well as a copy of its Validator Config. Each Validator Owner address must appear only once in the Validator Set. Validator Nodes run according to the configuration defined in the Validator Set. The Validator Config in the Validator Set may be an older version of the Validator Config than is currently stored under Validator Owner's account. Validator Set changes in either the set of participants or the Validator Config result in a Reconfiguration.
+* *Reconfiguration* defines a new epoch by emitting a `LibraConfig::NewEpochEvent`. This event is emitted as the result of any change in the global on-chain configuration. If a transaction causes an epoch change, that transaction is the last transaction committed in the current epoch — any subsequent transactions in that block or future blocks from that epoch will be ignored and available for later execution. The `LibraConfig::NewEpochEvent` marks the start of the new configuration computed as a result of the epoch ending transaction.
+* *LibraRoot* is the sole entity capable of adding or removing Validators and specifying the current Validator Set. It may also submit an AdminScript.
 * *AdminScript* is a transaction script that can be sent by the LibraRoot on behalf of any other account.
 
 <img alt="Validator Set" src={useBaseUrl('img/validator_set.png')} />
```

Comment:
Where does this image live? Can we not just embed it?

Reply:
images must be hosted somewhere, working with dfuzer on this